### PR TITLE
fix(git) mentions modern-era 'git switch' command instead of former 'git checkout'

### PR DESCRIPTION
**Pull request by dduportal**

### CODE REVIEW

File: content/chapitres/introduction-to-git.adoc [hunk 1]
```diff
@@ -258,7 +258,7 @@ image::scm-merge.png[caption="SCM Merge"]
 [source,bash]
 
 --
-git checkout master
+git switch master
```
Comment (jlevesy) [inline content/chapitres/introduction-to-git.adoc]: (Suggestion, non blocking): should we consider using main instead of master?  

```suggestion
git switch main
```

If yes, maybe let's do this in another PR :)

**Review by jlevesy — APPROVED** [review]
LGTM thanks!

File: content/chapitres/introduction-to-git.adoc (hunk 1 above)
Comment (dduportal) [reply content/chapitres/introduction-to-git.adoc]: good catch.
